upgrade/upgrade-scripts/upgrade-container

```diff
@@ -428,6 +428,12 @@ function migrate_file() {
 			die "'mkdir -p $parentdir' failed for '$CONTAINER'"
 		cp "$path" "/var/lib/machines/${CONTAINER}${path}" ||
 			die "'cp $path' failed for '$CONTAINER'"
+	else
+		# If a migration target file doesn't exist pre-upgrade then
+		# we delete the file. This case could happen if pre-upgrade
+		# configuration doesn't exist but a refresh install placed
+		# package default configuration.
+		rm -f "/var/lib/machines/${CONTAINER}${path}"
```

Comment:
add `|| die "failed to remove '$path' in '$CONTAINER'"` in case `rm` fails? since we're using `-f`, even if the file doesn't exist in the container, it shouldn't report an error.